framework/src/util/Utils.cpp

```diff
@@ -392,7 +392,8 @@ int GetLastErrorNo()
 std::string GetLastErrorStr()
 {
 #ifdef US_PLATFORM_POSIX
-  return std::string(strerror(errno));
+  char* errorString = strerror(errno);
```

Comment:
Yep, I saw this as well. Thanks for opening the issue. 
The thread-safe versions of strerror are not part of standard POSIX I believe. To support MacOS, we'll have to double check which versions of thread-safe strerror are supported.